src/libraries/System.IO.FileSystem.DriveInfo/src/System/IO/DriveInfo.cs

```diff
@@ -19,6 +20,7 @@ public DriveInfo(string driveName)
             _name = NormalizeDriveName(driveName);
         }
 
+        [UnsupportedOSPlatform("browser")]
```

Comment:
This is same as #41085. It does not make sense to be marking serialization infrastructure like this as unsupported. I think this PR can be closed.